modules/sso.php

```diff
@@ -220,8 +220,16 @@ public function login_body_class( $classes ) {
 		//Always add the jetpack-sso class so that we can add SSO specific styling even when the SSO form isn't being displayed.
 		$classes[] = 'jetpack-sso';
 
-		// If jetpack-sso-default-form, show the default login form.
-		if ( isset( $_GET['jetpack-sso-default-form'] ) && 1 == $_GET['jetpack-sso-default-form'] ) {
+		/**
+		 * Should we show the default or SSO login form?
+		 *
+		 * $_GET['jetpack-sso-default-form'] is used to provide a fallback in case JavaScript is not enabled.
+		 *
+		 * The default_to_sso_login() method allows us to dynamically decide which type of login to show. The SSO module
+		 * uses the method to display the default login form if we can not find a user to log in via SSO. But, the method
+		 * could be filtered by a site admin to always show the default login form if that is preferred.
+		 */
+		if ( ! empty( $_GET['jetpack-sso-default-form'] )  || ! Jetpack_SSO_Helpers::default_to_sso_login() ) {
```

Comment:
Thanks for the comment above. 
I find this really hard to follow. I don't know if returning true here will display the default form or display the sso form. 
My guess  is that if the statment is true that we should show the default form. 

I think to make it more clear we could do something like
```
$show_default_wp_login_form = ( isset( $_GET['jetpack-sso-default-form'] ) || ! Jetpack_SSO_Helpers::display_sso_login() )
if ( $show_default_wp_login_form ) {
return $classes;
}
```